### PR TITLE
adds a new bucket with an uzi and cmb revolver, and buffs and restats them

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
UZI is built to be used one handed and akimbo, use it to your hearts content. Renamed from GAL-9 since gal-9 sounds dumb, renamed to MP-2
CMB is tp-44 fire rate considering it's an autorevolver and fires revolver/small so it does less damage, also has a 2 round burst mode if you want to get cool or something.

## Why It's Good For The Game
Adds new bucket and makes shitty old guns viable again.

## Changelog
:cl:
expansion: new seasonal bucket with uzi and cmb revolver
balance: cmb now has tp-44 fire rate, now has a 2 round burst with 0.1 fire delay. Uzi buffed to fire as fast as normal smgs and is built for one handed use.
/:cl:

### CODE REVIEW

**Comment by David-Stormwell** [comment]
did you field test any of the guns to make sure they aren't overpowered?

**Comment by lbnesquik** [comment]
Where's the bucket

File: code/modules/projectiles/guns/revolvers.dm [hunk 1]
```diff
@@ -531,8 +531,8 @@
 	)
 	attachable_offset = list("muzzle_x" = 29, "muzzle_y" = 22,"rail_x" = 11, "rail_y" = 25, "under_x" = 20, "under_y" = 18, "stock_x" = 20, "stock_y" = 18)
 
-	fire_delay = 1.2 SECONDS
+	fire_delay = 0.15 SECONDS
```
Comment (SplinterGP) [inline code/modules/projectiles/guns/revolvers.dm:534]:
```suggestion
	fire_delay = 0.25 SECONDS
```

File: code/modules/projectiles/guns/smgs.dm [hunk 2]
```diff
@@ -324,10 +324,11 @@
...
-	fire_delay = 0.175 SECONDS
+	fire_delay = 0.15 SECONDS
 	burst_amount = 4
-	accuracy_mult_unwielded = 0.85
-	scatter = 15
-	scatter_unwielded = 60
+	accuracy_mult_unwielded = 0.9
+	accuracy_mult = 1
+	scatter = 0
+	scatter_unwielded = 10
```
Comment (SplinterGP) [inline code/modules/projectiles/guns/smgs.dm:332]:
```suggestion
	scatter = 10
	scatter_unwielded = 15
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/revolvers.dm (hunk 1 above)
Comment (Pariah919) [reply code/modules/projectiles/guns/revolvers.dm:534]: why should the cmb have more fire delay than a tp-44 anyway
i get i gave it 3 round burst memes with low fire delay but that's still less overall damage in the amount of time a tp-44 snubnose could output damage
It'd probably be better with 2 round burst and 0.2 though imo, and remove the damage mult that makes it do 1.5 more points of damage

File: code/modules/projectiles/guns/smgs.dm (hunk 2 above)
Comment (Pariah919) [reply code/modules/projectiles/guns/smgs.dm:332]: the uzi has nothing over a machine pistol other than 2 more bullets and better akimbo stats, don't see why it needs 10-15 scatter

**Pull request merged**